package.json

```diff
@@ -37,7 +37,7 @@
     "@leafygreen-ui/icon": "^9.0.0",
     "@leafygreen-ui/icon-button": "^9.1.2",
     "@leafygreen-ui/leafygreen-provider": "^2.0.2",
-    "@leafygreen-ui/tabs": "^3.0.1",
+    "@leafygreen-ui/tabs": "^5.1.2",
```

Comment:
For the version change we should install using `npm i @leafygreen-ui/tabs@5.1.2`, which updates the lockfile as well for us since that is also a record of all of our dependencies.

Reply:
Oh whoops just saw that change, please disregard.